src/utils/assertions.ts

```diff
@@ -1,16 +1,31 @@
 import { UnexpectedError } from './errors';
 
+export type Nullable<T> = T | null | undefined;
+
+/**
+ *  Returns true if the object is not null or undefined.
+ * @param object Object to check
+ * @returns True if the object is not null or undefined, false otherwise
+ */
+export function isNonNull<T>(object: Nullable<T>): object is T {
```

Comment:
What about this suggestion?

```suggestion
export function isDefined<T>(object: Nullable<T>): object is T {
```